Realm/RLMSyncUser.h

```diff
@@ -83,29 +83,26 @@ NS_ASSUME_NONNULL_BEGIN
 @interface RLMSyncUser : NSObject
 
 /**
- A dictionary of all valid, logged-in user identities corresponding to their user objects.
+ The unique Realm Object Server user ID string identifying this user.
```

Comment:
These references to ROS seem out of date?

Reply:
Going to remove all ROS references in a future PR.